commands/command.go

```diff
@@ -209,6 +212,8 @@ func (c *BaseCommand) run(args []string) error {
 		c.storage = storage.New(fileStrategy)
 	}
 
+	c.service = telemetry.NewService(c.flagTelemetryMode, c.user.APIKey, c.Name, c.UI)
```

Comment:
if this sets up the service, won't all of the START tracks fail?

Reply:
I think an alternative approach would be to create the service here as well as issue the START track call here

Something like: 
```
c.service = telemetry.NewService(c.flagTelemetryMode, c.user.APIKey, c.Name, c.UI)
dc.service.TrackEvent(telemetry.EventTypeCommandStart)